app/controllers/director/counselors_controller.rb

```diff
@@ -29,15 +29,15 @@ def create
       )
       if @counselor.save
         puts "COUNSELOR SAVED"
-        redirect_to counselor_path(:id => @counselor.id)
+        redirect_to counselors_path
```

Comment:
Had it direct to the index instead, thought it would be easier to see all the counselors once one was created. Though thinking about it now, it might be better to redirect to that particular counselor's page.